src/GPL-3.0.xml

```diff
@@ -14,14 +14,22 @@
     <standardLicenseHeader>
       Copyright (C)<alt name="copyright"
       match=".+">&lt;year&gt; &lt;name of author&gt;</alt>
-      This program is free software: you can redistribute it and/or
-      modify it under the terms of the GNU General Public License as
-      published by the Free Software Foundation, version. This program is
-      distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
-      without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
-      PARTICULAR PURPOSE. See the GNU General Public License for more details.
-      You should have received a copy of the GNU General Public License
-      along with this program. If not, see http://www.gnu.org/licenses/
+      <p>
+        This program is free software: you can redistribute it and/or
+        modify it under the terms of the GNU General Public License
+        as published by the Free Software Foundation, version 3.
```

Comment:
This is the fixed-to-match `GPL-3-only` version text.

There are a number of forms of this grant in the Linux kernel, and we probably want to support more of them.  But that is a bigger change than I wanted to bite off in this PR.